Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM openjdk:17
+VOLUME /tmp
+ARG JAR_FILE
+COPY build/libs/currency-rate-converter-2.0.0-SNAPSHOT.jar app.jar
```

Comment:
You are now using the fat jar to create the Docker container. What is a better strategy to create your app that will result in more efficient container downloads when deploying the application?